internal/cmd/cmd.go

```diff
@@ -46,7 +46,7 @@ func Root(cmd *cobra.Command, args []string) error {
 	defer s.Service.Delete(s.Dir)
 	defer s.DeleteCloudContainerImage()
 	if err != nil {
-		return fmt.Errorf("[cmd.Root] building and deploying sample to Cloud Run: %w", err)
+		return fmt.Errorf("Lifecycle.Execute building and deploying sample to Cloud Run: %w", err)
```

Comment:
nit: Usual format is "package.Function: [optional message]: %w", note the colon after the method name.

Reply:
Resolved, thank you.